app/views/advanced_search_finder/show.html.erb

```diff
@@ -55,13 +55,23 @@
       </div>
 
       <div class="govuk-grid-column-two-thirds">
-        <div class='js-live-search-results-block'>
+        <div class="js-live-search-results-block" data-module="remove-filter">
           <div class="filtered-results">
-            <div aria-live='assertive' id='js-search-results-info'>
-              <%= render_mustache('result_count', @results.to_hash) %>
+            <div class="result-info">
+              <h2 class="result-region-header__counter" id="js-result-count">
+                <%= number_with_delimiter(@results.total, :delimiter => ',') %> <%= @results.pluralised_document_noun %>
```

Comment:
You do the same thing in the ResultSetPresenter `to_hash` method for total. Could this be reused as `<%= @results.to_hash[:total] ...`?